students/kegan/session02/print_grid.py

```diff
@@ -0,0 +1,54 @@
+"""
+Kathryn Egan
+
+Prints a row x column grid consisting
+of this square:
+* - - - - *
+|         |
+|         |
+|         |
+|         |
+* - - - - *
+"""
+import argparse
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        'row', help='number of rows to print between 1 and 10',
+        type=int, choices=range(1, 11))
+    parser.add_argument(
+        'column', help='number of columns to print between 1 and 10',
+        type=int, choices=range(1, 11))
+    args = parser.parse_args()
+    # print row by column grid
+    for i in range(args.row):
+        print_horizontal(args.column)
+        for i in range(4):
+            print_vertical(args.column)
+    print_horizontal(args.column)
+
+
+def print_horizontal(column):
```

Comment:
Nice job breaking it out into functions.

But if you going to do that -/ a bit better to have the functions return a string, and do the printing when you call it. More flexible and extendable that way — say you wanted to write it to a file?